tests/e2e-opampbridge/opampbridge/00-install.yaml

```diff
@@ -14,7 +14,7 @@ spec:
     spec:
       containers:
         - name: e2e-test-app-bridge-server
-          image: ghcr.io/open-telemetry/opentelemetry-operator/e2e-test-app-bridge-server:ve2e
+          image: ghcr.io/open-telemetry/opentelemetry-operator/e2e-test-app-bridge-server:main
           ports:
```

Comment:
@IshwarKanse i need to do this so that we can use the bridge-server image built during CI rather than the one off of main. Is there a way to load the image into openshift?

Reply:
Ok, np. I'll update the OpenShift CI to build the image as well and use it in the job run. I'll update this PR removing this change.

Reply:
Done, removed the change.

Reply:
thank you :bow: